docs/changehistory/NextVersion.md

```diff
@@ -47,36 +50,64 @@ with:
 [Tile]($frontend)s are now required to report the [TileRequestChannel]($frontend) via which requests for their content should be executed, by implementing the new abstract [Tile.channel]($frontend) property. The channel needs to specify a name and a concurrency. The name must be unique among all registered channels, so choose something unlikely to conflict. The concurrency specifies the maximum number of requests that can be simultaneously active on the channel. For example, when using HTTP 1.1 modern browsers allow no more than 6 simultaneous connections to a given hostname, so 6 is a good concurrency for HTTP 1.1-based channels and the hostname is a decent choice for the channel's name.
 
 Typically all tiles in the same [TileTree]($frontend) use the same channel. Your implementation of `Tile.channel` will depend on the mechanism by which the content is obtained. If it uses HTTP, it's easy:
+
 ```ts
   public get channel() { return IModelApp.tileAdmin.getForHttp("my-unique-channel-name"); }
 ```
 
 If your tile never requests content, you can implement like so:
+
 ```ts
   public get channel() { throw new Error("This tile never has content so this property should never be invoked"); }
 ```
 
 If your tile uses the `alpha` `TileAdmin.requestElementGraphics` API, use the dedicated channel for such requests:
+
 ```ts
   public get channel() { return IModelApp.tileAdmin.channels.elementGraphicsRpc; }
 ```
 
 Otherwise, you must register a channel ahead of time. Choose an appropriate concurrency:
+
 - If the tile requests content from some custom [RpcInterface]($common), use `IModelApp.tileAdmin.channels.rpcConcurrency`.
 - Otherwise, choose a reasonably small limit to prevent too much work from being done at one time. Remember that tile requests are frequently canceled shortly after they are enqueued as the user navigates the view. A concurrency somewhere around 6-10 is probably reasonable.
 
 To register a channel at startup:
+
 ```ts
   await IModelApp.startup();
   const channel = new TileRequestChannel("my-unique-channel-name", IModelApp.tileAdmin.rpcConcurrency);
   IModelApp.tileAdmin.channels.add(channel);
 ```
 
 If you store `channel` from the above snippet in a global variable, you can implement your `channel` property to return it directly; otherwise you must look it up:
+
 ```ts
   public get channel() {
     const channel = IModelApp.tileAdmin.channels.get("my-unique-channel-name");
     assert(undefined !== channel);
     return channel;
   }
 ```
+
+### Authentication changes for Electron and Mobile apps
+
+For desktop and mobile applications, all authentication happens on the backend. The frontend process merely initiates the login process and waits for notification that it succeeds. Previously the steps required to set up the process was somewhat complicated.
```

Comment:
Inconsequential typo: "was" should be "were".

Reply:
oops, i already fixed this when i copied this text into the PR description. I'll fix here.

Reply:
fixed